WORKSPACE

```diff
@@ -29,3 +31,55 @@ com_google_gapic_generator_cpp_repositories()
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
+
+# WARNING: The java and go segments below are a hack to get
```

Comment:
What is the intended final state for this? Can we remove the hacks in this PR, or are we blocked?

Reply:
I don't know how permanent the hacks are. They are now tracked by #35